guides/source/upgrading_ruby_on_rails.md

```diff
@@ -81,6 +81,17 @@ Upgrading from Rails 7.1 to Rails 7.2
 
 For more information on changes made to Rails 7.2 please see the [release notes](7_2_release_notes.html).
 
+### `PostgresqlAdapter` now decodes dates automatically
+
+Manual queries with columns of type date will automatically be decoded to `Date` objects.
+To disable this new behavior and return dates as `String`:
+
+```ruby
+# config/application.rb
+config.active_record.postgresql_adapter_decode_dates = false
+```
+
+
```

Comment:
I don't think this is needed, because by default the config is `false`. It's only `true` if you enable it, and the comments in the new framework defaults file will cover that.